src/watertap_contrib/reflo/costing/units/air_stripping.py

```diff
@@ -0,0 +1,563 @@
+    tower_diam_in = pyo.units.convert(ax.tower_diam, to_units=pyo.units.inch)
+    flow_vol_air = pyo.units.convert(
+        pyo.units.convert(
+            prop_in.flow_vol_phase["Vap"],
+            to_units=pyo.units.m**3 / pyo.units.hr,
+        )
+        * pyo.units.hr
+        / pyo.units.m**3,
+        to_units=pyo.units.dimensionless,
+    )
+
+    base_currency = blk.config.flowsheet_costing_block.base_currency
+
+    # default packing material is PVC
+    if packing_material == "ceramic":
+        ax_params.capital_cost_packing.fix(2000)
+    if packing_material == "stainless_steel":
+        ax_params.capital_cost_packing.fix(8000)
+
+    blk.tower_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Aluminum tower cost",
+    )
+
+    blk.port_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Aluminum access port system cost",
+    )
+
+    blk.piping_liq_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Liquid inlet/outlet piping cost",
+    )
+
+    blk.piping_air_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Air inlet piping cost",
+    )
+
+    blk.tray_ring_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Tray ring cost",
+    )
+
+    blk.tray_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Tray cost",
+    )
+
+    blk.plate_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Plate cost",
+    )
+
+    blk.tower_internals_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Tower internals cost",
+    )
+
+    blk.packing_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Packing material cost",
+    )
+
+    blk.mist_eliminator_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Mist eliminator cost",
+    )
+
+    blk.pump_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Water pump cost",
+    )
+
+    blk.pump_power = pyo.Var(
+        initialize=50,
+        bounds=(0, None),
+        units=pyo.units.kilowatt,
+        doc="Water pump power requirement",
+    )
+
+    blk.blower_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Air blower cost",
+    )
+
+    blk.blower_power = pyo.Var(
+        initialize=50,
+        bounds=(0, None),
+        units=pyo.units.kilowatt,
+        doc="Air blower power requirement",
+    )
```

Comment:
I would prefer this to be on the unit model itself. There could be the case where the user might only care for energy-related modeling without inclusion of costs.